.github/ISSUE_TEMPLATE/socks-access-request.yml

```
@@ -31,6 +31,14 @@ body:
       placeholder: e.g. jane.doe@company.com
     validations:
       required: true
+  - type: input
+    id: github_handle
+    attributes:
+      label: Your Github Handle
+      description: Please provide your github handle
```

How about this? Clarifying it's their work one. 

```suggestion
      description: Please provide your work GitHub username
```